aws-lambda-java-log4j2/README.md

```diff
@@ -62,6 +62,33 @@ If using maven shade plugin, set the plugin configuration as follows
 </plugins>
 ```
 
+If you are using the [John Rengelman](https://github.com/johnrengelman/shadow) Gradle shadow plugin, then the plugin configuration is as follows:
+
+```groovy
+ 
+dependencies{
+  ...
+    implementation group: 'com.amazonaws', name: 'aws-lambda-java-log4j2', version: '1.1.0'
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: log4jVersion
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: log4jVersion
+    implementation group: 'org.apache.logging.log4j', name: 'log4j-slf4j18-impl', version: log4jVersion
```

Comment:
I don't think `log4j-slf4j18-impl` should be included in these docs as its usage with the library as it's not a required dependency to have.